layers/+intl/japanese/packages.el

```diff
@@ -0,0 +1,94 @@
+;;; packages.el --- Japanese Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Kenji Miyazaki <kenjimyzk@gmail.com>
+;; URL: https://github.com/kenjimyzk/
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;; List of all packages to install and/or initialize. Built-in packages
+;; which require an initialization must be listed explicitly in the list.
+(setq japanese-packages
+      '(evil-tutor-ja
+        migemo
+        avy-migemo
+        ddskk
+        japanese-holidays
+        pangu-spacing
+        org))
+
+(defun japanese/init-evil-tutor-ja ()
+  (use-package evil-tutor-ja
+    :defer t))
+
+(defun japanese/init-migemo ()
+  (use-package migemo
+    :defer t
+    :config
+    (setq migemo-command "cmigemo")
+    (setq migemo-options '("-q" "--emacs" "-i" "\a"))
+    (setq migemo-dictionary "/usr/local/share/migemo/utf-8/migemo-dict")
+    (setq migemo-user-dictionary nil)
+    (setq migemo-regex-dictionary nil)
+    (setq migemo-coding-system 'utf-8-unix)
+    (setq search-default-regexp-mode nil)
+    (migemo-init)
+    (japanese/post-init-helm-mode)))
+
+(defun japanese/post-init-helm-mode ()
```

Comment:
Please, don't use such names for functions that are not supposed to initialise packages listed in `japanese-packages`. Since that function is called once - you can just inline it where it's called. Otherwise - please define all helper functions in `funcs.el` file.